### ISSUE TITLE
Promise not analyzed?

### ISSUE THREAD

**Issue by alexnm** [amilajack/eslint-plugin-compat#97]
Hi, trying to integrate the plugin with a wider range of browsers to test stuff around and I noticed that `Promises` are not flagged with this setup

```
"browserslist": [
    "> 1%",
    "last 2 versions",
    "Firefox ESR",
    "ie 11"
  ],
```

Any idea if this should work? Or is there an exhaustive list of features that this plugin tests for?
Thanks

**Comment by nicolaisueper** [comment]
Same here for Promises and Arrow-Fn's with following configuration: 
```
ie >= 9
> 0.5%
last 2 versions
```
